qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -739,7 +739,12 @@ def on_load_finished(self, tab, ok):
             start = config.val.colors.tabs.indicator.start
             stop = config.val.colors.tabs.indicator.stop
             system = config.val.colors.tabs.indicator.system
-            color = utils.interpolate_color(start, stop, 100, system)
+
+            qurl = self.widget.tab_url(idx)
```

Comment:
Actually, one of the tests does suggest that this can be invalid, we shouldn't crash when that happens:

```
INVALID: Traceback (most recent call last):
INVALID:   File "/home/jay/Code/qutebrowser/qutebrowser/mainwindow/tabbedbrowser.py", line 761, in on_load_finished
INVALID:     qurl = self.widget.tab_url(idx)
INVALID:   File "/home/jay/Code/qutebrowser/qutebrowser/mainwindow/tabwidget.py", line 340, in tab_url
INVALID:     qtutils.ensure_valid(url)
INVALID:   File "/home/jay/Code/qutebrowser/qutebrowser/utils/qtutils.py", line 146, in ensure_valid
INVALID:     raise QtValueError(obj)
INVALID: qutebrowser.utils.qtutils.QtValueError: PyQt5.QtCore.QUrl('') is not valid
```